guides/micronaut-cloud-trace-base/create-app.adoc

```diff
@@ -0,0 +1,70 @@
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+=== OpenTelemetry
+
+The Micronaut framework uses https://opentelemetry.io/[OpenTelemetry] to generate and export tracing data.
+
+OpenTelemetry provides two annotations one to create a span and another to include additional information to the span.
+
+@WithSpan:: Used on methods to creates a new span using a given name or defaults to the method name.
+
+@SpanAttribute:: Used on method parameter to include the value to the span with a given name or defaults to the parameter name.
+
+The `@WithSpan` and `@SpanAttribute` can be used on non-private methods.
```

Comment:
```suggestion
`@WithSpan` and `@SpanAttribute` can be used on non-private methods.
```
? Based on the next sentence, I'm wondering if this should say "can be used **only** on non-private methods."